src/app.ts

```diff
@@ -183,9 +187,47 @@ export async function newApp(
   registerAPIRoutes(app, APIBasePath, allowNotifyIPSourceRange);
   registerPagoPARoutes(app, PagoPABasePath, allowPagoPAIPSourceRange);
 
+  const timer = idpMetadataUpdater(app);
+  app.on("server:stop", () => {
+    clearInterval(timer);
+  });
   return app;
 }
 
+export function idpMetadataUpdater(
+  app: Express,
+  listener?: () => void
+): NodeJS.Timer {
+  return setInterval(async () => {
```

Comment:
also add a log saying that the periodic update is starting

Reply:
logs [here](https://github.com/teamdigitale/io-backend/pull/425/files#diff-f98485da1849ae631ba5c04e5cec07ddR206) and [here](https://github.com/teamdigitale/io-backend/pull/425/files#diff-f98485da1849ae631ba5c04e5cec07ddR228) are not enough?